WordPress/Classes/ViewRelated/Stats/Charts/StatsBarChartView.swift

```diff
@@ -63,30 +63,43 @@ class StatsBarChartView: BarChartView {
     /// - Parameter entry: the selected entry for which to determine highlight information
     /// - Returns: the frame & offset from the bar that should be used to render the marker
     ///
-    private func calculateHighlightFrameAndOffset(for entry: ChartDataEntry) -> (CGRect, CGPoint) {
+    private func calculateMarkerFrameAndOffset(for entry: ChartDataEntry) -> (frame: CGRect, offset: CGPoint) {
```

Comment:
I renamed the method, in an attempt to mitigate confusion between the marker & highlight elements.